apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -953,3 +954,7 @@ func (dc *CassandraDatacenter) DatacenterName() string {
 	}
 	return dc.Name
 }
+
+func (dc *CassandraDatacenter) UseClientImage() bool {
```

Comment:
**suggestion:** Some comments or a different method name could help understanding the intent here. Currently I got confused by what this is supposed to support as feature (no mention of config building for example).

Reply:
True, I just lacked imagination when creating the method. Perhaps something like "NewConfigSystem() bool" ?

Reply:
I mean the real idea is to check if the deployed Cassandra version uses old Config.java or the newer one.

Reply:
`NewConfigSystem()` sounds better. Maybe `Builder` would be better than `System`?

The idea indeed is that with versions equal or higher than 4.1, we'll use the new config builder that sits in the k8ssandra-client project.
`New` is a little vague as well, because if we come up with another one later (god forbids) we'll end up with `NewNew` ? :D 
Is `UseK8ssandraConfigBuilder()` too long? You could also turn it into `K8ssandraConfigBuilder() bool`, which may work with Go idioms I guess?

Reply:
Ah, I didn't mean to reflect on the newer config-builder, I was thinking about from the perspective of Cassandra itself. That said, of course Cassandra's Config system can evolve as well later.